### PR TITLE
fix the name of custom objective function

### PR DESCRIPTION
**Pull request by guolinke**

### CODE REVIEW

**Comment by StrikerRUS** [comment]
@guolinke How this fix will help to use `objective='softmax'` (https://github.com/Microsoft/LightGBM/pull/1229#issuecomment-362950802)?

**Comment by guolinke** [comment]
@StrikerRUS okay, I see, you are talking about the alias of multiclass objective. I will add it.

**Comment by StrikerRUS** [comment]
@guolinke Thanks!
Sorry, for the misunderstanding.
